lib/beaker/host/unix.rb

```diff
@@ -64,16 +64,16 @@ def self.aio_defaults
       h = self.foss_defaults
       h['puppetserver-confdir'] = '/etc/puppetlabs/puppetserver/conf.d'
       h['puppetservice']  = 'puppetserver'
-      h['puppetbindir']   = '/opt/puppetlabs/agent/bin'
+      h['puppetbindir']   = '/opt/puppetlabs/puppet/bin'
```

Comment:
@joshcooper / @haus - better to use `/opt/puppetlabs/puppet/bin` or `/opt/puppetlabs/bin` which has symlinks?